src/element.ts

```diff
@@ -213,8 +225,9 @@ export abstract class Element {
   }
 
   /** Draw an element. */
-  // eslint-disable-next-line
-  abstract draw(...args: any[]): void;
+  draw(): void {
+    throw new RuntimeError('Element', 'Draw not defined');
```

Comment:
Since Element is no longer abstract, it would be nice if the methods didn't throw RuntimeErrors.

Maybe draw() needs some default implementation to draw an empty string at xShift=0, yShift=0?

Reply:
It is not the shifts, it is the `xPos` and `yPos`  required by `renderText'. The default implementation would draw all the elements at the same position, overlapping with each other.
If you think that it adds value, I can include with `xPos` and `yPos` set to 0. Should I?

Reply:
That's why I was asking about removing the `abstract` keyword. If you remove it, people can use the Element class directly, but it'll just throw errors. In this case, the error message is not super useful since it isn't telling us that xPos and yPos need to be set to a good position.

If you want Element to not be abstract, then I think draw() should have a default implementation. I'm OK with xPos and yPos being 0, and having the text element just drawn at 0,0 with default scale. Maybe that's useful for quick debugging. You could always console.warn() if you think drawing an element at 0,0 would be confusing to a developer. The warning could let the developer know that they should override the draw() method to something more useful.

Reply:
@rvilarl I'm going to merge this PR but we should continue discussing the possibility of having a default draw() method that does something (instead of throwing a RuntimeError).